aisdc/attacks/worst_case_attack.py

```diff
@@ -218,7 +220,7 @@ def run_attack_reps(  # pylint: disable = too-many-locals
         )
 
         mia_metrics = []
-        for rep in range(self.args.n_reps):
+        for rep in range(1, self.args.n_reps + 1, 1):
             logger.info("Rep %d of %d", rep, self.args.n_reps)
```

Comment:
i.e. here just change to 
`logger.info("Rep %d of %d", rep + 1, self.args.n_reps)`

Reply:
Agreed - will make changes accordingly